po/ca.po

```diff
@@ -1917,9 +1917,8 @@ msgid ""
 msgstr ""
 
 #: ../subiquity/ui/views/identity.py:212
-#, fuzzy
 msgid "Passwords"
-msgstr "contrasenyes"
+msgstr ""
```

Comment:
Are we fine deleting translations?

Reply:
I think the script is removing all the fuzzy translations.
Those are not used by the `gettext` during the runtime as far as I am aware.

Reply:
Thanks @liushuyu